pkg/virt-controller/watch/vm.go

```diff
@@ -2025,6 +1921,130 @@ func (c *VMController) processFailureCondition(vm *virtv1.VirtualMachine, vmi *v
 	return
 }
 
+func (c *VMController) isClearChangeRequestNeeded(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) bool {
```

Comment:
As commented next, `isTrimFisrtChangeRequestNeeded()` might be a more intuitive name for this function. Or `isClearFisrtChangeRequestNeeded()`